packages/frontend/src/components/DataViz/hooks/useTableDataModel.ts

```diff
@@ -1,7 +1,9 @@
 import {
     ChartKind,
     TableDataModel,
+    type IResultsRunner,
     type RawResultRow,
+    type VizChartLayout,
```

Comment:
shouldn't be needed

Reply:
Same for all files in the DataViz directory

Is the goal to fix this next?

Reply:
> Is the goal to fix this next?

Yes